src/DotNet.Testcontainers/Containers/TestcontainersContainer.cs

```diff
@@ -194,6 +200,7 @@ public Task<ExecResult> ExecAsync(IList<string> command, CancellationToken ct =
     /// <returns>A task that represents the asynchronous clean up operation of a Testcontainer.</returns>
     public async Task CleanUpAsync(CancellationToken ct = default)
     {
+      // TODO: We need at least one way to clean up Windows containers. ResourceReaper won't work.
```

Comment:
Do you know why the `ResourceReaper` won't work for Windows containers? Would it be possible to introduce a change into the Ryuk application so that Ryuk is also able to clean up Windows containers? Or would we need to provide Ryuk as a Windows container image? I think it would be nice to be able to also clean up Windows containers via a resource reaper container.

Reply:
Well, if there is no WSL available and the Windows engine is enabled then you can not pull and run Unix images - as far as I know. I'll need to double check that in the next days.

Reply:
ok, in that case it would probably be a good idea to create a custom Windows container ryuk image.

Reply:
Yep, I think that too. In the meantime we can keep the old clean up behaviour (without to purge orphan resources):

> // TODO: Either remove test or fix DisposeAsync (set ContainerListResponse; keep compatibility).

Then Windows containers get at least disposed (removed) if the test host does not crash.